test/coreSpec.js

```diff
@@ -434,17 +436,17 @@ describe('Core', function () {
         })
       })
     })
-    it.skip('returns a quote when there is a direct (but remote) path', function * () {
+    it('returns a quote when there is a direct (but remote) path', function * () {
```

Comment:
That's what https://github.com/interledgerjs/ilp-core/pull/89/commits/39beac6ad1d74483a359798200b1635565e1a762#diff-58f3a08270cc753a5a98228e9dcffb7aR367 tests, right? The route has no curve, so remote quoting is used.